src/array/without.ts

```diff
@@ -0,0 +1,15 @@
+export {}; // needed for TS to realize this file can be imported
+
+declare global {
+  interface Array<T> {
+    without(element: T): Array<T>;
+  }
+
+  interface ReadonlyArray<T> {
+    without(element: T): Array<T>;
+  }
+}
+
+Array.prototype.without = function (element) {
+  return this.filter((e) => e !== element);
```

Comment:
wonder if this would be more efficient

```suggestion
  const result = [...this];
  result.remove(element);
  return result;
```